packages/forms/docs/stories/UniversalFormModal.story.js

```diff
@@ -10,11 +21,65 @@ import {
     showAppNotification,
 } from '@rainbow-modules/app';
 import { useConnectModal, useOpenModal } from '@rainbow-modules/hooks';
-
 import app from '../../../../firebase';
 import UniversalFormModal from '../../src/components/UniversalFormModal';
 import isRequired from '../../src/validators/isRequired';
 
+const StyledBadge = styled(Badge)`
+    margin: 0 5px;
+`;
+
+const TableContainer = styled.div`
+    margin: 20px 50px;
+`;
+
+const TitleContainer = styled.div`
+    margin: 20px 50px 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+`;
+
+const Title = styled.h1`
+    font-size: 30px;
+    color: ${(props) => props.theme.rainbow.palette.text.main};
+    font-family: Lato Bold, Helvetica, sans-serif;
+`;
+
+const convertCetgoriesToString = (values) => {
+    const { categories } = values;
+    const newCategories = categories.reduce((accumulator, currentValue) => {
+        return [...accumulator, currentValue.label];
+    }, []);
+    const stringCategories = String(newCategories);
+    const newValues = { ...values, categories: stringCategories };
+    return newValues;
+};
+
+const convertCetgoriesToArray = (values) => {
+    const categoriesMap = {
+        Fantasy: 'option-1',
+        Adventure: 'option-2',
+        Classics: 'option-3',
+        Academic: 'option-4',
+        Novel: 'option-5',
+    };
+    const categories = values.categories.split(',');
+    const arrayCategories = categories.reduce((accumulator, currentValue) => {
+        return [...accumulator, { label: currentValue, name: categoriesMap[currentValue] }];
+    }, []);
+    const newValue = { ...values, categories: arrayCategories };
+    return newValue;
+};
```

Comment:
you should use the same value for the name of the option and the label because they are unique. it will save you the use on the `categoriesMap`

simple version
```js
const convertCategoriesToArray = (values) => {
    return {
        ...values,
        categories: values.categories.split(',').map(cat => ({ label: cat, name: cat })),
    }
}
```